### PR TITLE
test: expose a util method to enable large payload recording

### PR DESCRIPTION
**Pull request by yugangw-msft**
Per out discussion, @troydai, I am exposing a util method for tests to opt-in

### General Guidelines

- [na] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [na] Each command and parameter has a meaningful description.
- [na] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5568
This is an **experimental preview** for @microsoft users.

**Comment by yugangw-msft** [comment]
> I think it should be the other way around. You should opt in to the large response body processor, not opt-out.

I can go either way. @troydai ?

**Comment by tjprescott** [comment]
Fixes #4127.

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def test_storage_blob_upload_small_file(self, resource_group, storage_account):
     @ResourceGroupPreparer()
     @StorageAccountPreparer()
     def test_storage_blob_upload_midsize_file(self, resource_group, storage_account):
+        self.enable_payload_cap()
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py]: Verified it works, w/o is opt-out, the recording is 18MB

**Comment by yugangw-msft** [comment]
Close it and i am moving the change to devtools https://github.com/Azure/azure-python-devtools/pull/38

**Pull request closed without merging**